src/config/index.ts

```diff
@@ -27,6 +27,11 @@ export const config = convict({
     env: 'PORT'
   },
   env: {
+    format: ['development', 'test', 'pref test', 'production'],
```

Comment:
Watch out for the `pref test` typo

Think we should match them to the actual env names? [See comment](https://github.com/DEFRA/forms-designer/pull/483/files#r1825666648)

```suggestion
    format: ['dev', 'test', 'perf-test', 'prod'],
```

Reply:
corrected typo. I will wait for @alexluckett to confirm regarding env names, same for designer PR